### DISCUSSION TITLE
Picklist confirmation dialog

### DISCUSSION THREAD

**Discussion by garryatbrighton** [YetiForceCompany/YetiForceCRM#15562] [Feature improvement]
It would be of an advantage if system can support popup dialogs related to picklist. For instance changing status of picklist can be supported with various messages such as: Did you attached documets to the record? YES NO or Are you sure this record should be Closed? YES NO etc etc

**Comment by vovpff** [comment]
+1

**Comment by mariuszkrzaczkowski** [comment]
It works fine so it's designed to always be the sum of the whole, it doesn't work with filters.
Look:  Software configuration > System tools > Event handlers 
Tab "Validation before saving the record"
https://gitdeveloper.yetiforce.com/index.php?parent=Settings&module=EventHandler&view=Index
